charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.53.2
+version: 0.53.3
```

Comment:
Since this is a breaking change lets make this a 0.54.0

Reply:
done!